packages/scripts/src/helpers/scripts.ts

```diff
@@ -1493,7 +1500,30 @@ export async function createMinioSecret(k8sClient: K8s): Promise<void> {
     }
 }
 
-export async function getTerasliceImageFromConfigYaml(configFilePath: string): Promise<string> {
+export async function getConfigValueFromCustomYaml(
+    configFilePath: string,
+    valuePath: string
+): Promise<string> {
     const customConfig = yaml.load(fs.readFileSync(configFilePath, 'utf8')) as any;
-    return `${customConfig.teraslice.image.repository}:${customConfig.teraslice.image.tag}`;
+
+    const value = get(customConfig, valuePath, undefined);
+    if (!value) {
+        throw new Error(`Failed to find a config value ${valuePath} in the custom yaml at ${configFilePath}`);
+    }
+    return value;
+}
+
+export async function setConfigValuesFromCustomYaml(
```

Comment:
```suggestion
export async function setConfigValuesForCustomYaml(
```
Would this make more sense as a name?